R/dtm.R

```diff
@@ -1,24 +1,24 @@
-# @name get_dtm
-# @title Extract document-term matrix
-# @description This function extracts a document-term matrix from a
+#' @name get_dtm
```

Comment:
This was done intentionally. Since `get_dtm` is not available in public package API (not exported) we don't need to export documentation. So this roxygen-like docs are only for internal usage.